src/provider/model/StoredSettings.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * List of available settings in the settings collection.
+ */
+export const enum SettingsMap {
+    WINDOW_VISIBLE = 'windowVisible'
+}
+
+/**
+ * Shape of the internal settings stored in the database.
+ */
+export interface StoredSettings {
```

Comment:
Should this be singular? Like, it represents a single setting?

Reply:
Done!